roles/openshift_excluder/tasks/status.yml

```diff
@@ -20,13 +20,15 @@
 - name: Update to latest excluder packages
   package:
     name: "{{ openshift.common.service_type }}-excluder"
+    state: latest
```

Comment:
https://github.com/openshift/openshift-ansible/pull/3502/files#diff-b6b3dc3686358fd5aacff902dccb98a9R2 explains why i'd updated them in this task file.